app/views/planning_applications/assessment/informatives/index.html.erb

```diff
@@ -17,15 +17,15 @@
 <div class="govuk-grid-row" data-controller="informatives">
   <div class="govuk-grid-column-full">
     <% if @informative_set.informatives.select(&:persisted?).any? %>
-      <ul data-controller="sortable" class="govuk-list">
-        <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+      <ol data-controller="sortable" class="govuk-list">
```

Comment:
The content model for `<ul>` and `<ol>` models only allows `<li>` and script supporting elements (i.e. `<script>` and `<template>` elements).

https://html.spec.whatwg.org/multipage/grouping-content.html#the-ol-element